cogboard-webapp/src/components/BoardCard/index.js

```diff
@@ -59,13 +112,15 @@ const BoardCard = ({ boardData, className }) => {
               onClick={handleEditClick}
               aria-label="edit"
               size="small"
+              data-cy="board-card-edit-button"
```

Comment:
please be consistent with naming approach from your other PR:
`data-cy="edit-widget-cancel-button"`